src/main/java/whobot/utils/Helper.java

```diff
@@ -0,0 +1,168 @@
+package whobot.utils;
+
+import java.util.Arrays;
+import java.util.List;
+
+import whobot.main.UI;
+
+/***
+ * Class to Display all Help Messages
+ */
+public class Helper {
+
+    /** UI to which output should be sent */
+    private final UI ui;
+
+    /***
+     * Helper Class Constructor
+     * @param ui UI to which output should be sent
+     */
+    public Helper(UI ui) {
+        this.ui = ui;
+    }
+
+    /***
+     * Shows the Main Help Page
+     */
+    public void showMainHelp() {
+        String helpString = "These are the list of commands you can give me:\n";
```

Comment:
Maybe you can consider avoiding using `+=` for constructing the help message as that seems less obvious what the find result would be?